gcloud/pubsub/subscription.py

```diff
@@ -305,8 +305,10 @@ def set_iam_policy(self, policy, client=None):
         client = self._require_client(client)
         path = '%s:setIamPolicy' % (self.path,)
         resource = policy.to_api_repr()
+        # 'set_iam_policy' API requires an extra wrapper. :(
```

Comment:
Just an explanation:  the API *takes* a resource with the extra wrapper around the `Policy` resource, but *returns* a bare `Policy` resource, which is borked.